ciri/api/tests/test_find_transactions.c

```diff
@@ -8,8 +8,103 @@
 #include <unity/unity.h>
 
 #include "ciri/api/api.h"
+#include "ciri/api/tests/defs.h"
+#include "consensus/test_utils/bundle.h"
+#include "consensus/test_utils/tangle.h"
 
+static char *test_db_path = "ciri/api/tests/test.db";
+static char *ciri_db_path = "ciri/api/tests/ciri.db";
+static connection_config_t config;
 static iota_api_t api;
+static iota_consensus_t consensus;
+
+// Compares two queues by using an intermediary hash set
+static bool hash243_queue_cmp(hash243_queue_t const lhs,
+                              hash243_queue_t const rhs) {
+  hash243_set_t set = NULL;
+  hash243_queue_entry_t *iter = NULL;
+  bool cmp = true;
+
+  if (hash243_queue_count(lhs) != hash243_queue_count(rhs)) {
```

Comment:
What if a queue contains duplication/s?

Reply:
I'm actually using an intermediary set in this function

Reply:
What did you mean by "this function is not in the test only namespace" ? It actually is

Reply:
you're right